zerver/views/webhooks/github.py

```diff
@@ -40,6 +43,7 @@ def github_generic_content(noun, payload, blob):
 
 
 def api_github_v1(user_profile, event, payload, branches, stream, **kwargs):
+    # type: (UserProfile, text_type, Dict[text_type, Any], text_type, text_type,  **Any) -> Sequence(text_type, text_type, text_type)
```

Comment:
Why have you passed 3 params to `Sequence` (I'm curious why would you do it)? It just takes one type parameter afaik. And as pointed out by timabbott, you should be using square brackets.

Also remove the extra whitespace before ` **Any`.

You should also import names you are using in type comments (something like this `from typing import Dict, List, Sequence`). Otherwise mypy will give errors of the form `Name 'Sequence' is not defined`.